web/src/app/layout.tsx

```diff
@@ -70,11 +72,37 @@ export default async function RootLayout({
 
             <Card className="p-8 max-w-md">
               <h1 className="text-2xl font-bold mb-4 text-error">Error</h1>
-              <p className="text-text-500">
-                Your Danswer instance was not configured properly and your
-                settings could not be loaded. Please contact your admin to fix
-                this error.
-              </p>
+              {EE_ENABLED ? (
```

Comment:
I don't think this is used anymore (we should just remove it). What's goal for this change anyways?

Reply:
Ideally not have self-hosted users confused by the "please contact your Danswer admin to get it added." Can make  more general copy

Reply:
Oops, should have been more clear—`EE_ENABLED` isn't actually set anymore so this will always use the latter of the two options 😅

Also, EE_ENABLED isn't really the same as self-hosted vs cloud. Maybe we could just use the new message 100% of the time?